docs/about/whats-new.ipynb

```diff
@@ -1036,21 +1036,18 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "source": [
     "### Buffer and meta data access\n",
     "\n",
     "<div class=\"alert alert-info\">\n",
     "\n",
-    "**New in 0.6**\n",
-    "\n",
-    "</div>\n",
-    "\n",
-    "<div class=\"alert alert-info\">\n",
-    "\n",
     "**New in 0.7**\n",
     "\n",
     "- The internal buffer holding the \"events\" underlying binned data can now be accessed directly using the new `events` property.\n",
+    "  **Update: This is deprecated as of 0.8.2.**\n",
```

Comment:
Don't we have any release notes for patch releases? I don't see 0.7.1, nor 0.8.1 in the release notes.

Reply:
Yes I didn't bother with those so far: usually when something was broken no one except for us had noticed yet. Once we have more active users we will certainly have release notes for patches.